spec/models/post_action_spec.rb

```diff
@@ -501,6 +501,37 @@ def value_for(user_id, dt)
       expect(post.hidden_reason_id).to eq(Post.hidden_reasons[:flagged_by_tl3_user])
     end
 
+    it "hide non-tl4 posts that are flagged by a tl4 user" do
+      Discourse.stubs(:site_contact_user).returns(admin)
+
+      post_action_type = PostActionType.types[[:off_topic, :inappropriate, :spam].sample]
```

Comment:
definitely never add random to tests, just pick one of the 3, it is a nightmare to debug if this fails 33% of the time